docs/specs/parsing.md

```diff
@@ -13,18 +18,75 @@
     - `DateTime`, `DateTimeOffset`
     - `TimeSpan`
     - `Guid`
+    - `Uri`
 * Parses different cultures
     - e.g. English and German dates
```

Comment:
We should consider separating culture-dependent date-time parsing. I think it would be reasonable if it was handled by System.Text.Formatting library (which needs to be renamed but it is higher level text handling library using the primitives). The primitives library would handle UTC "R" format parsing only (which is needed for parsing networking protocols).